### PR TITLE
Fix crash that can occur when accessing media over XML-RPC

### LINKED ISSUE #19199: NSUnknownKeyException: [<__NSCFBoolean 0x23bba5c08> valueForUndefinedKey:]: this class is not key value coding-compliant...
Sentry Issue: [WORDPRESS-IOS-3MVY](https://sentry.io/organizations/a8c/issues/3483515760/?referrer=github_integration)

```
NSUnknownKeyException: [<__NSCFBoolean 0x23bba5c08> valueForUndefinedKey:]: this class is not key value coding-compliant for the key sizes.
  File "MediaServiceRemoteXMLRPC.m", line 276, in -[MediaServiceRemoteXMLRPC remoteMediaFromXMLRPCDictionary:]
  File "WPMapFilterReduce.m", line 9, in -[NSArray(WPMapFilterReduce) wp_map:]
  File "MediaServiceRemoteXMLRPC.m", line 253, in -[MediaServiceRemoteXMLRPC remoteMediaFromXMLRPCArray:]
  File "MediaServiceRemoteXMLRPC.m", line 57, in __86-[MediaServiceRemoteXMLRPC getMediaLibraryStartOffset:media:pageLoad:success:failure:]_block_invoke
  File "WordPressOrgXMLRPCApi.swift", line 166, in WordPressOrgXMLRPCApi.callMethod
...
(22 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by twstokes**
Fixes https://github.com/wordpress-mobile/WordPress-iOS/issues/19199

**Related PRs:**
- https://github.com/wordpress-mobile/WordPressKit-iOS/pull/537
- https://github.com/wordpress-mobile/WordPress-iOS/pull/18997

**To test:**
- Following the [testing steps here](https://github.com/wordpress-mobile/WordPressKit-iOS/pull/537).

## Regression Notes
1. Potential unintended areas of impact
    - Accessing media from the WordPress media library

2. What I did to test those areas of impact (or what existing automated tests I relied on)
    - Manual tests from the testing steps

3. What automated tests I added (or what prevented me from doing so)
    - None

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -47,10 +47,10 @@ def wordpress_ui
 end
 
 def wordpress_kit
-  pod 'WordPressKit', '~> 4.56.0'
+  # pod 'WordPressKit', '~> 4.56.0'
```
Comment (dcalhoun) [inline Podfile]: Friendly reminder: we'll want to pull in the relevant `WordPressKit-iOS@4.56.1` changes via this line once we merge and publish https://github.com/wordpress-mobile/WordPressKit-iOS/pull/537.

**Review by dcalhoun — APPROVED** [review]
🚀 LGTM. Approved via https://github.com/wordpress-mobile/WordPressKit-iOS/pull/537#pullrequestreview-1076030637.

**Comment by geriux** [comment]
@twstokes Thanks for updating the target of the PR and the WordPressKit tag!

I see there are conflicts and the `Podfile.lock` needs updating after the tag update. Once that's updated we could merge if all checks pass ✅

**Comment by twstokes** [comment]
> @twstokes Thanks for updating the target of the PR and the WordPressKit tag!
> 
> I see there are conflicts and the `Podfile.lock` needs updating after the tag update. Once that's updated we could merge if all checks pass ✅

Thanks @geriux for spotting that! I'll get them fixed now. 👍

**Review by geriux — APPROVED** [review]
LGTM! Thank you for working on this bug! Nice work 🚀

**Pull request merged**